tsl/test/expected/compression_ddl.out

```diff
@@ -1688,37 +1676,32 @@ SET enable_seqscan = off;
 EXPLAIN (costs off) SELECT device_id, count(*)
 FROM compression_insert
 GROUP BY device_id;
-                                                               QUERY PLAN                                                                
------------------------------------------------------------------------------------------------------------------------------------------
- GroupAggregate
+                                                            QUERY PLAN                                                             
+-----------------------------------------------------------------------------------------------------------------------------------
+ HashAggregate
    Group Key: _hyper_31_107_chunk.device_id
-   ->  Merge Append
-         Sort Key: _hyper_31_107_chunk.device_id
+   ->  Append
          ->  Custom Scan (DecompressChunk) on _hyper_31_107_chunk
                ->  Index Scan using compress_hyper_32_108_chunk__compressed_hypertable_32_device_id on compress_hyper_32_108_chunk
          ->  Custom Scan (DecompressChunk) on _hyper_31_109_chunk
                ->  Index Scan using compress_hyper_32_110_chunk__compressed_hypertable_32_device_id on compress_hyper_32_110_chunk
          ->  Custom Scan (DecompressChunk) on _hyper_31_111_chunk
                ->  Index Scan using compress_hyper_32_112_chunk__compressed_hypertable_32_device_id on compress_hyper_32_112_chunk
-         ->  Sort
-               Sort Key: _hyper_31_113_chunk.device_id
-               ->  Custom Scan (DecompressChunk) on _hyper_31_113_chunk
-                     ->  Index Scan using compress_hyper_32_114_chunk__compressed_hypertable_32_device_id on compress_hyper_32_114_chunk
-         ->  Sort
-               Sort Key: _hyper_31_113_chunk.device_id
-               ->  Seq Scan on _hyper_31_113_chunk
-(17 rows)
+         ->  Custom Scan (DecompressChunk) on _hyper_31_113_chunk
+               ->  Index Scan using compress_hyper_32_114_chunk__compressed_hypertable_32_device_id on compress_hyper_32_114_chunk
+         ->  Index Only Scan using _hyper_31_113_chunk_compression_insert_device_id_time_idx on _hyper_31_113_chunk
+(12 rows)
 
 SELECT device_id, count(*)
 FROM compression_insert
 GROUP BY device_id;
```

Comment:
Those queries are missing ORDER BY clause making them indeterministic and potentially leading to flaky tests.

Reply:
Oh sorry I just realized that I did not address this comment... :( I will create a follow-up PR for this